tests/ui/imports/ambiguous-4.rs

```diff
@@ -5,5 +5,4 @@ extern crate ambiguous_4_extern;
 
 fn main() {
     ambiguous_4_extern::id();
-    // `warning_ambiguous` had been lost at metadata.
```

Comment:
It appears that `﻿ambiguous_4_extern::id` should be considered an ambiguous item, however, this isn't currently implemented in #114682.

Reply:
Is it due to this case https://github.com/rust-lang/rust/pull/114682#discussion_r1455745546 or something different?

Reply:
In any case, if this should be an ambiguity, then the same FIXME comment as in other tests would be appropriate.